### PR TITLE
Product collection example

### PR DESCRIPTION
**Pull request by Geosynopsis**

### CODE REVIEW

**Review by matthewhanson — APPROVED** [review]

File: product/examples/ProductCollectionOneatlas.json [hunk 1]
```diff
@@ -0,0 +1,467 @@
...
+      "keywords": ["optical", "oneatlas", "tasking"],
+      "license": "proprietary",
+      "providers": [
+        {
+          "name": "oneatlas",
+          "description": "oneatlas description",
+          "roles": ["producer"],
+          "url": "https://intelligence.airbus.com/"
+        },
+        {
```
Comment (CloudNiner) [inline product/examples/ProductCollectionOneatlas.json:27]: i think you can remove these up42 providers?

**Review by CloudNiner — APPROVED** [review]

File: product/examples/ProductCollectionOneatlas.json [hunk 2]
```diff
@@ -0,0 +1,467 @@
+{
+  "conformsTo": [
+    "https://stat-api.example.com/v0.1.0/core",
+    "https://stat-api.example.com/v0.1.0/opportunities"
+  ],
```
Comment (m-mohr) [inline product/examples/ProductCollectionOneatlas.json:5]:
```suggestion
```

File: product/examples/ProductCollectionOneatlas.json [hunk 3]
```diff
@@ -0,0 +1,463 @@
+{
+  "products": [
+    {
+      "type": "Product",
+      "stac_version": "1.0.0",
+      "stac_extensions": [],
```
Comment (m-mohr) [inline product/examples/ProductCollectionOneatlas.json:6]:
```suggestion
```

File: product/examples/ProductCollectionOneatlas.json (hunk 1 above)
Comment (Geosynopsis) [reply product/examples/ProductCollectionOneatlas.json:21]: Give an example of why the providers are an array of objects.

**Pull request merged**